src/subapps/dataExplorer/DatasetCount.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import * as pluralize from 'pluralize';
+import { isNil } from 'lodash';
+import './styles.less';
+
+interface Props {
+  nexusTotal: number;
+  totalFiltered?: number;
+  totalOnPage: number;
+}
+
+export const DatasetCount: React.FC<Props> = ({
+  nexusTotal,
+  totalOnPage,
+  totalFiltered,
+}: Props) => {
+  return (
+    <div className="data-explorer-count">
+      <span>
+        Total:{' '}
+        <b>
+          {nexusTotal.toLocaleString()} {pluralize('dataset', nexusTotal ?? 0)}
```

Comment:
I think the clearer format is to show the large number with separator so the user can read the number
you can use built-in js fn 
```
new Intl.NumberFormat(locale, { 
      maximumSignificantDigits: 3,
}).format(Number(nexusTotal))}
where locale is 'de-CH' or any other locale
```

Reply:
The thing is `Number.prototype.toLocaleString` calls Intl.NumberFormat internall ([MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString)) and it has a slimmer api so I went with that. I don't mind changing it to `Intl.NumberFormat` though

I think I should not pass the `maximumSignificantDigits` option because it round off the numbers:

```javascript
new Intl.NumberFormat('en-US', { 
      maximumSignificantDigits: 3,
}).format(Number(1234))
>> 1,230
```

The CH locales all look funny to me. For example, for the number 1343234325:

```javascript
new Intl.NumberFormat('fr-CH').format(Number(1343234325)) # "1 343 234 325"
new Intl.NumberFormat('de-CH').format(Number(1343234325)) # "1’343’234’325" 
```

Therefore, I am planning to use `en-US`:
```javascript
new Intl.NumberFormat('en-US').format(Number(1343234325)) # "1,343,234,325" 
```

Reply:
yeah `toLocaleString` call the` Intl.NumberFormat` but does not change the format, do I miss smthg ?
and for the rounding is not a problem for me, most statistics value are showed with rounding
![Screenshot 2023-07-11 at 13 39 37](https://github.com/BlueBrain/nexus-web/assets/2632473/65150b28-8af6-4cec-b739-3c63af3ec70d)
for the locale, yeah looks funny, we just need to choose one, and stick with it across the app, i use `de-CH` in my data page, but you can change it as you prefer.

Reply:
1. O yeah, a locale should be passed to `toLocaleString()`:
```
(121243).toLocaleString("de-CH")  # "121’243"
(121243).toLocaleString("en-US")  # "121,243"
```

2. I really think we should not round off the numbers. (If there are 1234 results, I shouldn't show there were only 1230 results returned from nexus), but that's just my opinion. If you want I can double check with Sam/Loris.

Reply:
ahh okey, so we can use it without rounding